assets/src/edit-story/components/library/panes/media/mediaElement.js

```diff
@@ -231,7 +220,7 @@ const MediaElement = ({
       >
         <Image
           key={src}
-          src={imageSrc}
+          src={useSmallestImage(resource)}
```

Comment:
React hooks can not be called conditionally.

Reply:
It's actually not a hook, see https://github.com/google/web-stories-wp/pull/1733#discussion_r429816020

It needs to be renamed.